projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/NormalSTIRFuturesProviderInterface.java

```diff
@@ -11,28 +11,28 @@
  * Provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike/delay dependent. 
```

Comment:
Javadoc: The volatility is expiration/delay/strike/futures price dependent.
Line length<120.